solidity/safemultisig/README.md

```diff
@@ -205,7 +199,7 @@ Use the following command to list the transactions currently awaiting custodian
 If there are some transactions requiring confirmation, they will be displayed.
 
 ### 4.6 Creating transaction online
-An internet connection is not required to create a signed transaction message. Use to following command to do it:
+Use the following command to create a new transaction:
```

Comment:
Lets note that new transaction is queued in multisig and is waiting for necessary amount of confirmations from other custodians before it will be executed.